### ISSUE TITLE
Invalid URL in the path, should start with a domain

### ISSUE THREAD

**Issue by liaochangjian** [GitbookIO/gitbook#2336]
## Bug description

when I Start my local development server. and Then open the space in your web browser, using http://localhost:3000/.

prompt : Invalid URL in the path, should start with a domain

I can't enter the main pages.

help me please.

**Comment by addisonschultz** [comment]
Hi @liaochangjian - this could definitely be documented a bit more. You'll need to include the published space url (of a space you have permissions in) after the local version - for instance, you could put something like: 

http://localhost:3000/docs.gitbook.com

**Comment by POMXARK** [comment]
> Hi @liaochangjian - this could definitely be documented a bit more. You'll need to include the published space url (of a space you have permissions in) after the local version - for instance, you could put something like:
> 
> http://localhost:3000/docs.gitbook.com

how to make this space, that is, a convenient cms for updating content is not publicly available?

**Comment by addisonschultz** [comment]
@POMXARK - you'll still need to publish the space, but you could do so using a private/share link!

**Comment by edsiper** [comment]
what's the proper way to render my "local" gitbook content by using the server locally ?

**Comment by addisonschultz** [comment]
Hey @edsiper The content needs to be published in some way first - afterwards, you can direct traffic to the local version as so: https://github.com/GitbookIO/gitbook?tab=readme-ov-file#set-up

For instance: 

> Then open the space in your web browser, using http://localhost:3000// (example: http://localhost:3000/docs.gitbook.com).

**Comment by edsiper** [comment]
@addisonschultz, how will the service know where my local GIT repository exists in my filesystem ?

**Comment by addisonschultz** [comment]
@edsiper It's linked to the published version - meaning the editing experience doesn't change (online in the editor, or through GitSync) 

TLDR; it's linked to the published part, not the content part

**Comment by hoshiizumiya** [comment]
There is no local installation guide available, I totally dont konw how to make it possiabe to creat custom page.
Or it seems no longer support after gitbook cilent was deprecated. We can only edit them via gitbook webapp?

**Comment by edsiper** [comment]
I filed issue "OSS User Experience" #2419 to discuss local. offline mode and others

**Comment by addisonschultz** [comment]
Closing this as #2419 contains the most up to date information on fixing the developer experience of this repo
